docs-v2/content/en/schemas/v4beta13.json

```diff
@@ -1766,6 +1766,18 @@
             "[\"golang:1.10.1-alpine3.7\", \"alpine:3.7\"]"
           ]
         },
+        "cacheTo": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array",
+          "description": "the Docker images used as cache destination. If omitted, cacheFrom is used with max mode to export all layers.",
```

Comment:
Not terribly important, since most people playing with this (today at least) will already know what buildx `--cache-to` accepts, but I wouldn't mind a link to the docker docs here.

Reply:
Ok, will add a link to the docs: https://docs.docker.com/reference/cli/docker/buildx/build/#cache-to  (I see ko and other builders have similar links)